desc/plotting.py

```diff
@@ -2253,6 +2254,11 @@ def plot_coils(coils, grid=None, fig=None, return_data=False, **kwargs):
 
     def flatten_coils(coilset):
         if hasattr(coilset, "__len__"):
+            # plot all coils for symmetric coil sets
+            if coilset.NFP > 1 or coilset.sym:
```

Comment:
do we want to leave the option for only plotting unique coils?

Reply:
I think no? We only keep track of the unique coils to simplify the optimization, but in reality there are coils in every field period (since the magnetic field calculation assumes that). And typically when visualizing the coils you are interested in how the coils are spaced relative to each other, which needs the full coil set. So I can't think of a common use case when you would want only the unique coils and think it's best to always plot the "real" coil set

Reply:
Ok but I think this logic will break if you pass in a list of coils instead of a coilset, since it will have length but no NFP/sym attributes

Reply:
Hmm true, I'll fix that and a add a test

Reply:
I could imagine showing a figure that shows only the unique coiils in a coilset along with the rest of the plasma just to emphasize that it is only a few unique coils repeated, I have seen these before in past coil papers like for NCSX or in the regcoil paper.